### PR TITLE
add stack button on empty board

### PR DESCRIPTION
**Pull request by jakobroehrl**
Signed-off-by: Jakob Röhrl <jakob.roehrl@web.de>
* Resolves: https://github.com/nextcloud/deck/issues/1655
* Target version: master 

It shows only on empty board up:
grafik

### CODE REVIEW

**Comment by juliushaertl** [comment]
I think we should have this in a emptycontent layout here similar to the board not found message. You can use our new component from `@nextcloud/vue` for that: https://nextcloud-vue-components.netlify.app/#/Components/EmptyContent

**Comment by jakobroehrl** [comment]
> 
> I think we should have this in a emptycontent layout here similar to the board not found message. You can use our new component from `@nextcloud/vue` for that: https://nextcloud-vue-components.netlify.app/#/Components/EmptyContent

How to import it?
This is not working: 
 - import { EmptyContent } from '@nextcloud/vue'
 - import { EmptyContent } from '@nextcloud/vue/dist/Components/EmptyContent'

Why is here that way implemented?
https://github.com/nextcloud/deck/blob/master/src/components/board/Board.vue#L42-L46

**Comment by juliushaertl** [comment]
Requires https://github.com/nextcloud/nextcloud-vue/pull/1096 to be released first ;)

**Comment by juliushaertl** [comment]
Coming with https://github.com/nextcloud/deck/pull/2042

**Comment by juliushaertl** [comment]
@jakobroehrl master is now on `@nextcloud/vue@2.1.0` so the empty content component is ready to be used :)

**Comment by jakobroehrl** [comment]
@juliushaertl 
Sorry, it's not working: "Unknown custom element: <EmptyContent> - did you register the component correctly?"

**Comment by juliushaertl** [comment]
Well you did not rebase this branch yet, it still shows 

        "@nextcloud/vue": "^1.5.0",

File: src/components/board/Board.vue [hunk 1]
```diff
@@ -23,6 +23,26 @@
 <template>
 	<div class="board-wrapper">
 		<Controls :board="board" />
+
+		<EmptyContent v-if="stacksByBoard.length === 0" icon="icon-pause">
+			No lists, add one
```
Comment (juliushaertl) [inline src/components/board/Board.vue:28]: Should be translatable

Comment (juliushaertl) [reply src/components/board/Board.vue:28]: cc @jancborchardt for some nicer text maybe ;)

File: src/components/board/Board.vue [hunk 2]
```diff
@@ -117,11 +140,10 @@ export default {
 			this.$store.dispatch('orderStack', { stack: this.stacksByBoard[removedIndex], removedIndex, addedIndex })
 		},
 
-		createStack() {
+		addNewStack() {
```
Comment (juliushaertl) [inline src/components/board/Board.vue:143]: Make sure that newStackTitle is emptied once the stack has been created.

**Comment by jakobroehrl** [comment]
done, let's merge

File: src/components/board/Board.vue [hunk 3]
```diff
@@ -23,6 +23,26 @@
 <template>
 	<div class="board-wrapper">
 		<Controls :board="board" />
+
+		<EmptyContent v-if="stacksByBoard.length === 0" icon="icon-pause">
+			{{ t('deck', 'No lists, add one') }}
```
Comment (juliushaertl) [inline src/components/board/Board.vue:28]: Let's do something more descriptive then here like:

**No lists available**
Create a new list to add cards to this board

**Pull request merged**